pkgs/development/python-modules/zest_releaser/default.nix

```diff
@@ -0,0 +1,52 @@
+{ buildPythonPackage
+, colorama
+, fetchFromGitHub
+, lib
+, requests
+, setuptools
+, six
+, twine
+, test_setup
+, wheel
+, z3c_testsetup
+, zope_testing
+, zope_testrunner
+}:
+
+buildPythonPackage rec {
+  pname = "zest_releaser";
+  version = "6.22.1";
+
+  src = fetchFromGitHub {
+    owner = "zestsoftware";
+    repo = "zest.releaser";
+    rev = version;
+    sha256 = "005xmpwzkyg0q89vi485nhjjzi34yj2lpszigcph7y4p1bfzgiv4";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    colorama
+    requests
+    six
+    twine
+  ];
+
+  checkInputs = [
+    test_setup
+    wheel
+    z3c_testsetup
+    zope_testing
+    zope_testrunner
+  ];
+
+  pythonImportsCheck = [ "zest.releaser" ];
+
+  meta = with lib; {
+    description = "A collection of command-line programs to help automating the task of releasing a Python project";
+    downloadPage = "https://pypi.org/project/zest.releaser/";
```

Comment:
You don't need to set it, it is optional and mostly used for proprietary packages but if you already set it we can keep it.